src/frontend/src/btc/components/fee/UtxosFeeContext.svelte

```diff
@@ -20,7 +20,7 @@
 			return;
 		}
 
-		if (isNullish(networkId) || isNullish(amount) || amount === 0) {
+		if (isNullish(networkId) || isNullish(amount) || Number(amount) === 0) {
```

Comment:
Good question actually, is zero acceptable?

Just noticed we have a utils `invalidAmount` which asserts amount is now lower than zero and which seems used in various places.

Reply:
In this particular case - not acceptable as we need to make the value is not 0 because the `utxos` call fails if we request it with `amount = 0`.

Reply:
Thanks for the feedback. Can you add a comment about it?

Reply:
Yep, added.